spec.md

```diff
@@ -398,6 +398,34 @@ Example error response (invalid `id` provided):
 
 ### Resource Data Model
 
+A resource is a named (addressable) blob stored in a Space, with metadata.
```

Comment:
```suggestion
A resource is a named (addressable) object stored in a Space, with metadata.
```

This is where we have disagreed. I would like there to be many blobs associated with a name.

Reply:
(i.e. a resource may have many representations)

Reply:
Sure. changing.